src/components/List/ListSection.tsx

```diff
@@ -13,7 +14,9 @@ export type ListSectionProps = {
   rowSize?: ListRowSize;
 } & SectionProps<unknown>;
 
-const ListSection: React.FCC<ListSectionProps> = (props) => <AriaSection {...props} />;
+const ListSection: React.FC<React.PropsWithChildren<ListSectionProps>> = (props) => (
```

Comment:
this could probably be FCC

Reply:
resolved here https://github.com/Orfium/orfium-ictinus/pull/720/commits/f50892c7c98ff1d524fd9d7c6393c46fc9782161